typespec/Applications.Core/gateways.tsp

```diff
@@ -107,6 +107,9 @@ model GatewayRoute {
 
   @doc("Optionally update the prefix when sending the request to the service. Ex - replacePrefix: '/' and path: '/myservice' will transform '/myservice/myroute' to '/myroute'")
   replacePrefix?: string;
+
+  @doc("Enables websocket support for the route. Defaults to false.")
+  enableWebsockets?: boolean;
```

Comment:
Does it automatically default to false or does it need to be explicitly set like [this](https://github.com/radius-project/radius/blob/1c40cf3be4d12374740337bbd6f41c6cb9d59d03/typespec/Applications.Core/gateways.tsp#L53)?